docker_mcp/tools/containers.py

```
@@ -1021,12 +1233,28 @@ async def list_host_ports(self, host_id: str) -> dict[str, Any]:
             )
 
         except (DockerCommandError, DockerContextError) as e:
-            logger.error("Failed to list host ports", host_id=host_id, error=str(e))
+            logger.error("Failed to list host ports", host_id=host_id, error=str(e), error_type=type(e).__name__)
```

Ruff (0.14.4)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)